udata/settings.py

```diff
@@ -56,6 +56,34 @@ class Defaults(object):
 
     OAUTH2_PROVIDER_ERROR_ENDPOINT = 'oauth-i18n.oauth_error'
 
+    MD_ALLOWED_TAGS = [
+        'a',
+        'abbr',
+        'acronym',
+        'b',
+        'br',
+        'blockquote',
+        'code',
```

Comment:
Can you add a test combining `<code>` and urls within? I'm pretty sure urls will be linkified within it and that's not what we expect.